ota_pal.h

```diff
@@ -32,6 +32,14 @@
 #define OTA_PAL_H_
 
 #include "ota.h"
+#include "ota_appversion32.h"
```

Comment:
This header file can be removed now.

Reply:
Removed. But added it back to ota_pal.c for appFirmwareVersion.